design-reviews/recipes/engagement-process.md

```diff
@@ -0,0 +1,129 @@
+1. You cannot clearly illustrate all the tasks required for a story
+
+> **Tip:** After your sprint planning is complete your team should consider hosting an initial design review discussion to dive deep on the design requirement of the stories that were identified.  After you have more clarity the team should continue forward with either a sync or async design review to complete the tasks.
+
+## Sprint Backlog Grooming
+
+If you team is not already hosting a Sprint Backlog Grooming session at least once per week you should consider it.  It is a great opportunity to:
+
+1. Keep your backlog clean
+1. Re-prioritize work bash on shifting business priorities
+1. Fill in missing descriptions / acceptance criteria
+1. Identify stories that require design reviews
+
+The team can follow the same steps from [sprint planning](#sprint-planning) to help identify which stories require design reviews.  This can often save much time during your actual sprint planning meetings to focus on the task at hand.
+
+## Sprint Retrospectives
+
+Sprint retrospectives are a great time to check-in with your dev team to identify what is working or not working within your group and propose changes to keep improving.
+
+It is also a great time to check-in on your design reviews
+
+- Did any of your designs change from last sprint?
+- How have your design changes impacted your engagement?
+- Have previous design artifacts been updated to reflect new changes?
+
+All design artifacts including the game plan should be treated as a living document.  As requirements change or you uncover more unknowns the dev crew should be going back and updating all your design artifacts to reflect reality.  If this step is missed you are making a critical mistake which could lead to the customer incurring more technical debt in the future. Artifacts that are not up to date are basically `bugs` in your design.
+
+> **Tip:** Keep your artifacts up to date by adding it to your teams `Definition of Done` for all user stories.
+
+## Wrap-up Sprints
+
+Wrap-up sprints are a great time to tie up loose ends with your customer, hand-off your solution and execute your sharing requirements from the game plan.  Customer hand-off becomes a lot easier when you have design artifacts to reference and deliver along side your completed solution.  
+
+During your wrap-up sprints your dev crew should consider the following:
+
+1. Are your design artifacts up to date?
+1. Are your design artifacts stored in an accessible location?
+1. How can other CSE dev crews leverage the work from your engagement?
+1. Submit new engineering patterns and practices back to the [CSE playbook](https://aka.ms/cseplaybook)
+
+It is critical that your design artifacts are in an accessible location that can be shared by other CSE dev crews.  [CSE artifact hub](https://aka.ms/cseartifacthub) is a great place to store and share your artifacts so that other teams can quickly find them.

```

Comment:
Maybe this link shouldn't be included since it is only accessible internally?

Reply:
Good point - will review